app/cdap/components/RecipeList/RecipeTableRow.tsx

```diff
@@ -165,13 +165,17 @@ export const RecipeTableRow = ({
 
   return (
     <>
-      <div className="grid-row" onClick={!showAllColumns ? selectRecipeHandler : undefined}>
+      <div
+        className="grid-row"
+        onClick={!showAllColumns ? selectRecipeHandler : undefined}
```

Comment:
can we just say
`!showAllColumns && selectRecipeHandler `

Reply:
removed undefined